--- libbpf-bootstrap-4999ed86de09cdd75d9ca982d36419bb3de296c8_orig/src/Makefile	2020-12-17 06:32:05.000000000 +0000
+++ libbpf-bootstrap-4999ed86de09cdd75d9ca982d36419bb3de296c8/src/Makefile	2020-12-26 04:29:19.657956636 +0000
@@ -9,7 +9,7 @@
 CFLAGS := -g -Wall
 ARCH := $(shell uname -m | sed 's/x86_64/x86/')
 
-APPS = minimal bootstrap uprobe
+APPS = minimal bootstrap 
 
 # Get Clang's default includes on this system. We'll explicitly add these dirs
 # to the includes list when compiling with `-target bpf` because otherwise some
